lib/mox.ex

```diff
@@ -763,6 +763,11 @@ defmodule Mox do
     verify_mock_or_all!(self(), mock, :test)
   end
 
+  def get_executed_calls(mock) do
+    validate_mock!(mock)
+    Mox.Server.get_executed_calls(mock)
```

Comment:
Also, is this safe to execute concurrently? It doesn't look like it is the case. :(

Reply:
It definitely isn't 😉 Will have to figure it out with the allowance.